java/core/src/test/java/com/google/protobuf/DecodeUtf8Test.java

```diff
@@ -55,29 +55,24 @@ public void testTwoBytes() throws Exception {
   }
 
   public void testThreeBytes() throws Exception {
-    // Travis' OOM killer doesn't like this test
-    if (System.getenv("TRAVIS") == null) {
-      int count = 0;
-      int valid = 0;
-      for (int i = Byte.MIN_VALUE; i <= Byte.MAX_VALUE; i++) {
-        for (int j = Byte.MIN_VALUE; j <= Byte.MAX_VALUE; j++) {
-          for (int k = Byte.MIN_VALUE; k <= Byte.MAX_VALUE; k++) {
-            byte[] bytes = new byte[]{(byte) i, (byte) j, (byte) k};
-            ByteString bs = ByteString.copyFrom(bytes);
-            if (!bs.isValidUtf8()) {
-              assertInvalid(bytes);
-            } else {
-              valid++;
-            }
-            count++;
-            if (count % 1000000L == 0) {
-              logger.info("Processed " + (count / 1000000L) + " million characters");
-            }
+    int count = 0;
+    int valid = 0;
+    for (int i = Byte.MIN_VALUE; i <= Byte.MAX_VALUE; i++) {
+      for (int j = Byte.MIN_VALUE; j <= Byte.MAX_VALUE; j++) {
+        for (int k = Byte.MIN_VALUE; k <= Byte.MAX_VALUE; k++) {
+          byte[] bytes = new byte[]{(byte) i, (byte) j, (byte) k};
+          ByteString bs = ByteString.copyFrom(bytes);
+          if (bs.isValidUtf8()) {
+            valid++;
+          }
+          count++;
+          if (count % 1000000L == 0) {
```

Comment:
Random idea: do we also want to get rid of 16 million modulus and compare operations here? I don't know. What does profiling suggest? These log messages don't seem hugely useful except in the rare case when someone is watching the test output in real time. 

Also, why is the int count being modded with a long?

Reply:
Good point, this is probably OK to remove. I think it was useful for tracking the progress since the test was pretty slow, but now it should be fast enough to make this unnecessary. I'm not sure why the int was being modded with a long.